### PR TITLE
Feature/structural unification changes for docs

### PR DESCRIPTION
**Pull request by localredhead**

### CODE REVIEW

File: docs/guides/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Server-side Credentials
```
Comment (zachferland) [inline docs/guides/tutorial.md]: is `docs/guides/server-side-credentials-example.md` not a rewrite of this file?

File: package.json [hunk 2]
```diff
@@ -53,8 +53,9 @@
     "body-parser": "^1.17.1",
     "express": "^4.15.2",
     "jest": "^22.4.3",
-    "jsdoc": "^3.4.3",
-    "jsdoc-to-markdown": "^1.3.7",
+    "jsdoc": "^3.5.5",
+    "jsdoc-babel": "^0.4.0",
```
[the remark this answers is not in the document]
Comment (localredhead) [reply package.json]: Was working with it while trying to improve the output of the jsdoc.  Removed.

File: docs/guides/tutorial.md (hunk 1 above)
Comment (localredhead) [reply docs/guides/tutorial.md]: Yes.  Removing.

**Comment by localredhead** [comment]
@zachferland I cherry picked my changes to a clean rebase of develop to address this so I will close this PR for this one https://github.com/uport-project/uport-js/pull/66

**Pull request closed without merging**